### ISSUE TITLE
Allow me to update labels from list view without accessing the modal

### ISSUE THREAD

**Issue by adro718** [codetree/feedback#199]
Previous versions of code tree allowed me to change a user or a label without opening the full dialog. I was also able to copy the github link without opening the dialog. Anyway to get that functionality back into the list view?

**Comment by TimonVS** [comment]
I agree that changing a label without opening the modal would be a great feature. You can ctrl/command + click on an issue to open the issue on Github in a new tab.

**Comment by kareem** [comment]
Hi @adro718 @TimonVS haven't forgotten about this.  I agree that being able to change users or labels from the list view would be useful.  It's on our list.

**Comment by kareem** [comment]
@adro718 @TimonVS Just launched Assignee and Milestone editing from the List view.  Labels are on the drawing board.  Thanks for your patience!
